test/models/verify.ts

```diff
@@ -0,0 +1,393 @@
+import { ValidationError } from 'xrpl-local/common/errors'
+import { verify } from './../../src/models/transactions/verify'
+import { assert } from 'chai'
+
+/**
+ * Transaction Verification Testing
+ *
+ * Providing runtime verification testing for each specific transaction type
+ */
+describe('Global Transaction Verification', () => {
+
+    it (`verifies valid AccountDelete`, () => {
```

Comment:
Nest tests that are for the same transaction type in another `describe`

Reply:
What you could do for the tests is to just add `verify` to each of the existing tests, and expect the same result as the specific `verify` functions.

Reply:
Added verify to all!